types/bunyan-slack/index.d.ts

```diff
@@ -0,0 +1,29 @@
+// Type definitions for bunyan-slack 0.0
+// Project: https://github.com/qualitybath/bunyan-slack#readme
+// Definitions by: Eric Chen <https://github.com/fshiori>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+/// <reference types="node" />
+
+declare namespace BunyanSlack {
```

Comment:
I'm not familiar with `bunyan` or `bunyan-slack`. Is there a reason you have written the definition with both a global namespace and an ambient module declaration? If not, it would be much simpler to define the types in the following way:

```ts
declare class BunyanSlack {
  constructor(options: BunyanSlack.Options, error?: any);
  write(record: unknown): void;
}
declare namespace BunyanSlack {
  interface Options { ... }
}
export = BunyanSlack;
```